css/style.css

```diff
@@ -104,6 +108,89 @@ body {
   padding-top: 2.2rem;
 }
 
+/* DETAIL PAGE  */
+
+.popup {
+  display: none;
+  flex-direction: column;
+  justify-content: center;
+  position: sticky;
+  background: #fff;
+  top: 0;
+  padding: 1rem 1rem;
+  margin: 2rem 0.8rem;
+  border-radius: 10px;
+  height: 100%;
+  z-index: 2;
```

Comment:
- [ ] While you did make the changes requested, the new design has hidden the buttons. kindly fix this 👍 

# View:
![image](https://user-images.githubusercontent.com/50990925/216190529-165002ae-b16c-4dfe-9621-1c7ea123c7da.png)